ubtue_Brill.js

```diff
@@ -97,7 +97,7 @@ function postProcess(doc, item) {
 	}
 
 	//scrape ORCID from website
-	let authorSectionEntries = doc.querySelectorAll('.text-subheading span:nth-child(2)');
+	let authorSectionEntries = doc.querySelectorAll('.text-subheading span');
```

Comment:
Is the previous `:th-child`-approach really no longer valid for _all_ Brill journals? Otherwise testing for the second case if the first is empty might be a more defensive approach.